build-logic/convention/src/main/kotlin/AndroidHiltConventionPlugin.kt

```diff
@@ -30,10 +31,10 @@ class AndroidHiltConventionPlugin : Plugin<Project> {
             }
 
             dependencies {
-                "implementation"(libs.findLibrary("hilt.android").get())
-                "kapt"(libs.findLibrary("hilt.compiler").get())
-                "kaptAndroidTest"(libs.findLibrary("hilt.compiler").get())
-                "kaptTest"(libs.findLibrary("hilt.compiler").get())
+                implementation("hilt.android")
+                "kapt"(findLibrary("hilt.compiler"))
```

Comment:
I didn't write an extension for `Kapt` and `Ksp` because we use them only once. But I'm not sure, I would like to get your opinion on this. In this case, we have to make the `findLibrary` function external. If we write extension for `Kapt` and `Ksp`, we can make `findLibrary` function private.

Reply:
any idea? @javadude